### DISCUSSION TITLE
what is member node?

### DISCUSSION THREAD

**Discussion by scalalang2** [ConsenSys/quorum#1360] [Q&A] [answered]
According to [qourum-kubernetes](https://github.com/ConsenSys/quorum-kubernetes/tree/master/dev)

We should set up 4 validator node and 1 non-validator node (called member).
In this case. If I configured only 4 validator node. It doesn't propose block at all.

If I add 1 non-validator node. Now they start to propose block.

**Question here.**
1) Why it works like this? My guess is that IBFT consensus doesn't need non-validator node.
2) What is the member node? I cannot found that any explanation.

**Comment by antonydenyer** [comment, marked as answer]
You should probably start here https://consensys.net/docs/goquorum/en/latest/tutorials/quorum-dev-quickstart/getting-started/

A member node is simply someone who is a member of the network but does not not have authority to validate blocks (IBFT is a proof of authority consensus algorithm).

**Comment by scalalang2** [comment]
Thanks it gives me help a lot! : )

**Reply by ghost** [reply]
I have gone through the "quorum-dev-quickstart" example but all validator nodes are similar as member nodes. No any difference is there like configuration is same. So my question is that how we can differentiate both type of nodes:
1.  When we generate the keys using quorum genesis tool -
    command: npx quorum-genesis-tool --consensus qbft --chainID 1337 --blockperiod 5 --requestTimeout 10 --epochLength 30000 --difficulty 1 --gasLimit '0xFFFFFF' --coinbase '0x0000000000000000000000000000000000000000' --validators 4 --members 0 --bootnodes 0 --outputPath 'artifacts'

Question: 
   1. Can we use Validator node for transaction which we are doing using non-validator nodes?
   2. I have to use two quorum node setup using QBFT So what will the values which will pass using the above quorum-genesis-tool command? 
   3. how many validators will be required?
   4. And who will be the owner of validators and who will be the control authority of these validators -- for Exp: suppose one new node wants to join our network and want to add their own validator So in this case what will be the steps to add a new node.??z
   5. And still need to know how Validator nodes are different from non-validator nodes because as we know there is no any difference in configuration???????????????
   
Please help me to understand this concept.

Thanks in advance!

**Reply by antonydenyer** [reply]
> Can we use Validator node for transaction which we are doing using non-validator nodes?

Yes, the only additional responsibility that 'validators' have is that they are allowed to create blocks. 

> I have to use two quorum node setup using QBFT So what will the values which will pass using the above quorum-genesis-tool command?
> how many validators will be required?

4 is the minimum

> And who will be the owner of validators and who will be the control authority of these validators -- for Exp: suppose one new node wants to join our network and want to add their own validator So in this case what will be the steps to add a new node.??z

https://consensys.net/docs/goquorum/en/latest/tutorials/private-network/adding-removing-qbft-validators/

> And still need to know how Validator nodes are different from non-validator nodes because as we know there is no any difference in configuration???????????????

You can view the current validators using https://consensys.net/docs/goquorum/en/latest/reference/api-methods/#istanbul_getvalidators then if your nodes is in the list you're a validator!
